### DISCUSSION TITLE
When will CMS kit  be in Blazor WASM?

### DISCUSSION THREAD

**Discussion by sturlath** [abpframework/abp#11631] [Q&A]
Is there a plan for it?

**Comment by enisn** [comment]
CMS Kit targets MVC currently. CMS-like websites are used for public content such as blogs, forums, media galleries, etc.

All of those must be indexed by search-engines properly. So, we try to avoid using spa technologies for those kinds of content. 

We can develop admin side of CMS Kit for Blazor when it's required but there is no such a plan for public-side of CMS Kit

**Reply by sturlath** [reply]
Yes I was only talking about the admin part. My use of multitenancy is probably little bit different than normally.
 
1. I have a Blazor backend where the Host manages every major functionality of the tenants.
2. The tenants have their own backend with their specific stuff. They never user CMS.
3. I have a public web page that displays content from all tenants. This is where my CMS content is.

So I would like the Host to have access to the CMS controls. Now I have put them into the identity management service so I manage.. but having them in Blazor makes the most sense.
